lightweightlibrary/src/main/java/com/tradingview/lightweightcharts/api/serializer/TimeRangeDeserializer.kt

```diff
@@ -16,7 +17,7 @@ class TimeRangeDeserializer: Deserializer<TimeRange>() {
         val jsonObject = json.asJsonObject
         val from = jsonObject.get("from")
         val to = jsonObject.get("to")
-
+        Log.d("TimeRangeDeserializer","json: ${json}")
```

Comment:
[suggestion] later on may add static flag or use build type to enable or disable the logging